src/WinUI/ArcGISRuntime.WinUI.Viewer/MainPage.xaml.cs

```diff
@@ -81,12 +79,12 @@ private void LoadTreeView(SearchableTreeNode fullTree)
             }
             CategoriesTree.RootNodes.Clear();
 
-            muxc.TreeViewNode rootNode;
```

Comment:
Was this intended for this PR?

Reply:
This is some minor cleanup I did while editing this file. That import and prefix are not needed in WinUI, I believe they were in UWP.